ppr-ui/src/composables/mhrInformation/useMhrInformation.ts

```diff
@@ -298,7 +298,8 @@ export const useMhrInformation = () => {
             return owner.individualName ? { ...owner, individualName: normalizeObject(owner.individualName) } : owner
           }),
           // Determine group tenancy type
-          type: ownerGroup.owners.filter(owner => owner.action === ActionTypes.REMOVED).length > 1
+          type: (ownerGroup.owners.filter(owner => owner.action === ActionTypes.REMOVED).length > 1 ||
+            getMhrTransferType.value?.transferType === ApiTransferTypes.SURVIVING_JOINT_TENANT)
```

Comment:
After looking into the API and mhr_api code, it seems that for for SJT transfers we need to have JOINT type for the deleted owner to have successful submission. @doug-lovett @cameron-eyds

Reply:
Sounds good although may be worth checking out the matrix: https://docs.google.com/spreadsheets/d/1crIN1kjmva_64540WmMbyTV1BXj6FjCc/edit#gid=1836270235

I believe* there was intention to assign it an NA type but my memory is a little fuzzy on it.

Reply:
For the delete owner group, the type should always match the tenancy type returned by the API from the GET call.

Reply:
Hmm, ok, I will double-check that, thanks.

Reply:
@doug-lovett the GET call you are referring is the one before the POST call, right? In case of transfer submission, the GET request is `other-registrations/${registrationNum}`, but I do not see tenancy type in the response?

<img width="559" alt="Screen Shot 2023-05-09 at 1 31 07 PM" src="https://github.com/bcgov/ppr/assets/2333290/872534ce-0386-46f4-884a-2976fad43218">

That leads me to think that the GET request you are talking about is the one when we load the registration via `/mhr/api/v1/registrations/150411?current=true`, which has the tenancy type. In this case, in order to be able to file an SJT Transfer, we need to have at least two owners in the Group, which is always JOINT?

Reply:
Yes, the GET request I am referring to is the one where the UI fetches the current state of the MH. In this case it is 
GET /mhr/api/v1/registrations/150411?current=true. It returns all the active groups, and each group contains a tenancy type. For groupId=1, here is what is returned:
    {
      "groupId": 1,
      "interest": "UNDIVIDED",
      "interestDenominator": 6,
      "interestNumerator": 1,
      "owners": [
        {
          "address": {
            "city": "LANGLEY",
            "country": "CA",
            "postalCode": "V3A 6H4",
            "region": "BC",
            "street": "6665 238TH STREET"
          },
          "individualName": {
            "first": "MARY-ANNE",
            "last": "BICKNELL"
          },
          "ownerId": 1,
          "partyType": "OWNER_IND",
          "phoneNumber": "6044620279",
          "type": "JOINT"
        },
        {
          "address": {
            "city": "NORTH SAANICH",
            "country": "CA",
            "postalCode": "V8L 5V4",
            "region": "BC",
            "street": "123-1640 ELECTRA BLVD"
          },
          "individualName": {
            "first": "BRYAN",
            "last": "HOLMES"
          },
          "ownerId": 2,
          "partyType": "OWNER_IND",
          "phoneNumber": "6044620279",
          "type": "JOINT"
        }
      ],
      "status": "ACTIVE",
      "tenancySpecified": true,
      "type": "JOINT"
    },
Whenever deleting an owner group, the groupId and type must match was is in the database to pass data validation.
If the group contains 2 or more owners it can be JOINT or NA if it contains an EXECUTOR/TRUSTEE/ADMINISTRATOR.